notes/CodeFlag_notes.csv

```diff
@@ -138,3 +138,7 @@ noteID,note
 141,"See Appendix XYZ."
 142,"In the ring ordering scheme, the 12 rhomboids are not considered and the scanning mode applies to the grid in its entirety."
 143,"In the nested ordering scheme, the 12 rhomboids are scanned in order defined in Gorsky et al. 2015. The flag bits then refer to how the points are scanned within a rhomboid."
+225a,"The following additional arguments must be specified: • Circle - 1 argument for the radius in metres • Rectangle - 2 arguments for the length in 1. west-east and 2. south-north in metres • Square - 1 argument for the length of the equal-length sides in metres • Wedge - 3 arguments for 1. radius in metres and 2. start and 3. end radius in arithmetic degrees with 0 on the positive axis along west-east and counted counter-clockwise.  • span of grid cells - 2 arguments regarding the number of grid cells in 1. west-east and 2. south-north so that the span is i+/-x and j+/-y"
+225b,"The option quantile needs two additional arguments, 1. The total number of quantiles and 2. the quantile value"
+225c,"0/191/5-7 are intended to be used with templates 4.121 and 4.122 in which the spatiotemporal criteria is encoded used to get a categorical yes/no per grid point of each ensemble member"
+225d,"Scale Factor of Lower Limit, Scaled Value of Lower Limit, Scale Factor of Upper Limit and Scaled Value of Upper Limit must be set to missing. This entry can be used with 0/191/5-7 entries but are not limited to them"
```

Comment:
I'm not sure this makes sense..... 
Is this note associated with code table 4.9?

Reply:
which one of them? We have notes for different tables

Reply:
225d

Reply:
@sebvi this note still doesn't make sense to me.